README.md

```diff
@@ -75,19 +101,24 @@ func NewConfig() *Config
 
 #### New
 
-`New` creates an instance of your module. It must return an `io.Closer` and an `error.
+`New` creates an instance of your module. It must return an `io.Closer` and an `error`.
 The function takes a `context.Context`, the configuration structure returned by `NewConfig`,
 `Info` and `UI` objects. `Info` and `UI` are located in `github.com/glasslabs/looking-glass/module/types`.
```

Comment:
Here I'd suggest removing the last sentence and linking to the godoc of your own package. WDYT?